R/attributes.R

```diff
@@ -99,6 +101,44 @@ setAttributes <- function(item, attribs){
 }
 
 
+#' Function setAttribute
+#'
+#' Set one or more attributes on an object.  You can use this to add attribute
+#' annotation(s) to a DGEobj or to a specific item within a DGEobj. It will add
+#' the attribute but not delete any existing attributes, unlike the base attribute
```

Comment:
It looks like it will overwrite an existing attribute, this is important to point out.  You CAN remove an attribute by using this - the attr function if you have a value of NULL will remove the attribute.  I am not sure what you mean by the "base attribute function" in the unlike (I don't like documentation that says things like this, it's a personal pet peeve).  Let's fix this up to be consistent.  attr should also use exact = TRUE in the call, and allow NULL for the attrib value.  Also, use this function in setAttributes as the functional function, this keeps it maintainable.